src/webui/api/searchcontroller.cpp

```diff
@@ -388,7 +388,7 @@ QJsonArray SearchController::getPluginsInfo(const QStringList &plugins) const
         pluginsArray << QJsonObject
         {
             {u"name"_qs, pluginInfo->name},
-            {u"version"_qs, QString(pluginInfo->version)},
+            {u"version"_qs, static_cast<QString>(pluginInfo->version)},
```

Comment:
???

Reply:
IMO `QString(pluginInfo->version)` looks like invoking QString constructor for `Version` type which does not exist. I changed to `static_cast` to signify it is invoking `operator QString()` from `Version` class instead of others.

Reply:
This code looks confusing, IMO. I would implement `IStringable` by `Version` class and use `toString()` instead.

Reply:
>I would implement IStringable by Version class and use toString() instead.

OK, I'll revert this and do it in a later PR.